java/src/jmri/jmrit/logixng/implementation/DefaultNamedTableManager.java

```diff
@@ -149,7 +159,11 @@ public AnonymousTable newAnonymousTable(int numRows, int numColumns)
     public NamedTable loadTableFromCSVData(
```

Comment:
Please change this interface to throw IOException instead of throwing an RuntimeException in this method.